frontend/src/v5/ui/controls/ellipsisMenu/ellipsisMenu.component.tsx

```diff
@@ -15,78 +15,44 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { useState, cloneElement, MouseEvent } from 'react';
 import { formatMessage } from '@/v5/services/intl';
-import { ClickAwayListener, Tooltip, MenuList } from '@mui/material';
-import { EllipsisButton } from '@controls/ellipsisButton';
-import { Popover } from './ellipsisMenu.styles';
+import { Tooltip, MenuList } from '@mui/material';
+import EllipsisIcon from '@assets/icons/outlined/ellipsis-outlined.svg';
+import { ActionMenu } from '@controls/actionMenu';
+import { EllipsisButton } from './ellipsisMenu.styles';
 
 export interface IEllipsisMenu {
 	selected?: boolean;
-	children: JSX.Element[];
+	children: any;
 	className?: string;
+	disabled?: boolean;
 }
 
-export const EllipsisMenu = ({ selected, children, className }: IEllipsisMenu): JSX.Element => {
-	const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
-
-	const handleClickDropdown = (event: MouseEvent<HTMLDivElement>) => {
-		event.stopPropagation();
-		event.preventDefault();
-		setAnchorEl(event.currentTarget);
-	};
-
-	const handleCloseDropdown = () => {
-		setAnchorEl(null);
-	};
-
-	const handleListKeyDown = (event) => {
-		if (event.key === 'Tab') {
-			event.preventDefault();
-			handleCloseDropdown();
-		}
-	};
+export const EllipsisMenu = ({ selected, children, className, disabled }: IEllipsisMenu): JSX.Element => {
+	if (disabled) {
+		return (
+			<EllipsisButton variant={selected ? 'secondary' : 'primary'} disabled={disabled}>
```

Comment:
Yeah, I'd rather have the `disabled` option on the tooltip instead of the empty string too tbh. We could do that, but I would not that in this branch as I feel a few components could use the new approach.

The reason why I created the if is both the tooltip and the actionMenu, which in the first case is not required.
I think this is cleaner.